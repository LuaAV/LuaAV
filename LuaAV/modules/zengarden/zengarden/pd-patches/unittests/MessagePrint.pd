#N canvas 657 346 454 211 10;
#X obj 16 5 loadbang;
#X msg 158 111 walk the dog;
#X msg 82 111 bang;
#X msg 123 111 234;
#X obj 124 155 print x1;
#X obj 47 155 print -n;
#X msg 272 111 3.14;
#X msg 48 111 1;
#X msg 328 111 test;
#X obj 328 155 print;
#X obj 48 49 t b b b b b b;
#X obj 16 26 t b b;
#X connect 0 0 11 0;
#X connect 1 0 4 0;
#X connect 2 0 4 0;
#X connect 3 0 4 0;
#X connect 6 0 4 0;
#X connect 7 0 5 0;
#X connect 8 0 9 0;
#X connect 10 0 7 0;
#X connect 10 1 2 0;
#X connect 10 2 3 0;
#X connect 10 3 1 0;
#X connect 10 4 6 0;
#X connect 10 5 8 0;
#X connect 11 1 10 0;
