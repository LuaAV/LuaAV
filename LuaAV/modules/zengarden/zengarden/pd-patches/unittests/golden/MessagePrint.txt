print: test
x1: 3.14
x1: walk the dog
x1: 234
x1: bang
1
closing audio...
closing MIDI...
... done.
